Game/tr/Game.po

```diff
@@ -1747,231 +1747,231 @@ msgstr "Devam edin,ve iyi günler"
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,CallOut1"
 msgid "“I’ve been working at Phil’s for over 10 years, and they just let me go. It’s heartless.”"
-msgstr ""
+msgstr "Ben Phille 10 yıl üzerinde çalışıyorum ve benim gitmeme izin verdiler. Bu vicdansızlık."
 
 #. Key:	Headline
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,Headline"
 msgid "ARQUE CORPORATION COMPLETES ACQUISITION OF PHILLIP'S PHARMACEUTICALS"
-msgstr ""
+msgstr "ARQUE ŞİRKETİ PHİLLİP'İN İLAÇLARINI ALMAYI TAMAMLADI"
 
 #. Key:	Headline2
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,Headline2"
 msgid "SHIPS \"DANGEROUSLY CLOSE\" TO COLLISIONS WITHOUT LIGHTHOUSE"
-msgstr ""
+msgstr "GEMİLER\"TEHLİKELİ DERECEDE YAKIN\DENİZ FENERİ OLMADAN BİRBİRLERİYLE ÇARPIŞABİLİR"
 
-#. Key:	Headline3
+#. KeY	Headline3
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,Headline3"
 msgid "RED ALGAE NUMBERS STRONG"
-msgstr ""
+msgstr "KIRMIZI YOSUN GÜÇLÜLÜĞÜ SİMGELER"
 
 #. Key:	Navigation1
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,Navigation1"
 msgid "MORE ON PAGE 11"
-msgstr ""
+msgstr "SAYFA 11 DE DAHA FAZLA"
 
 #. Key:	Paragraph01
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,Paragraph01"
 msgid "The acquisition of local pharmaceutical company Phillip's Pharmaceuticals was completed late last week, as all fifty employees were let go, twenty more workers than originally planned."
-msgstr ""
+msgstr"Yerel ilaç şirketi Phillip's Pharmaceuticals'ın satın alınması geçtiğimiz hafta sonunda tamamlandı, çünkü elli çalışanın tamamı serbest bırakıldı, başlangıçta planlanandan yirmi daha fazla işçi çalıştı."
```

Comment:
Looks like this lost a space after `msgstr` and between the first " (I think it's important for the formatting)

Reply:
Looks like this is still an issue